src/dvclive/live.py

```diff
@@ -338,11 +346,24 @@ def make_report(self):
 
     def end(self):
         self.make_summary(update_step=False)
-        if self._studio_url and self._studio_token:
-            if "done" not in self._studio_events_to_skip:
-                if not post_to_studio(self, "done"):
-                    logger.warning("`post_to_studio` `done` event failed.")
-                self._studio_events_to_skip.add("done")
+        if "done" not in self._studio_events_to_skip:
+            response = False
+            if post_live_metrics is not None:
```

Comment:
if dvc_studio_client is not installed then it will fail silently. Maybe add a logger.warning("post_to_studio` `done` event failed - dvc_studio_client package is not installed"?

Reply:
I added it to the `start` event (you could not really reach the `done` event without the package installed).